scripting.md

```diff
@@ -0,0 +1,104 @@
+# SuperTux Level Editor Scripting Tutorial
+
+## What is Scripting?
+
+Scripting in SuperTux is when a game object is altered using things like a script trigger or a switch.  Scripting is written in the Squirrel coding language.
+
+#### What sort of things use scripting?
+
+Scripts can be activated by the following things, plus many more:
+
+* Script Trigger, which activates the script.
+* Switch/Button, which activate scripts when used.
+* Death Scripts, which activate when an enemy dies.
+* Init-Scripts, which activate when Tux enters the sector.
+
+In order to edit a script, right click a scriptable object and open the script editor.
+
+## Structure of scripts
+
+The basic structure of a line of script is as so:
+
+    object.action(Number);
```

Comment:
Please use the way of adding a code block with three ticks, like:

    ```
    code goes here
    ```

That is easier to recognize as a code block, and doesn't need indentation. (And do so everywhere, not just here.)